tests/fib/test_fib.py

```diff
@@ -1,19 +1,19 @@
 import pytest
-from netaddr import *
 import time
 import json
 import logging
-import requests
 from ptf_runner import ptf_runner
 from datetime import datetime 
 
 logger = logging.getLogger(__name__)
 
-HASH_KEYS = ['src-ip', 'dst-ip', 'src-port', 'dst-port', 'ingress-port']
+HASH_KEYS = ['src-ip', 'dst-ip', 'src-port', 'dst-port', 'ingress-port', 'src-mac', 'dst-mac', 'ip-proto', 'vlan-id']
```

Comment:
usually src-mac, dst-mac, vlan-id are optional hash keys. Not all the platform supports these optional hash keys. I think we need to provide options for the user to choose to test which hash key.

Reply:
a simple fix is to not enable these three by default. ideally, I think we should break each hash field into one test. and then we can use @pytest.mark.skip to skip running the tests on certain platform (long term)

Reply:
> a simple fix is to not enable these three by default. ideally, I think we should break each hash field into one test. and then we can use @pytest.mark.skip to skip running the tests on certain platform (long term)

Not enable src-mac, dst-mac and vlan-id hash keys by default.